addons/l10n_ro/partner_view.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
-	<data>
-		<record model="ir.ui.view" id="res_partner_form_ro">
-			<field name="name">res.partner.form.ro</field>
-			<field name="inherit_id" ref="base.view_partner_form"/>
-			<field name="model">res.partner</field>
-			<field name="arch" type="xml">
-				<field name="property_account_position" position="after">
-					<field name="nrc" placeholder="e.g. J12/1234/2012" class="oe_inline"/>
+    <data>
+        <record model="ir.ui.view" id="res_partner_form_ro">
+            <field name="name">res.partner.form.ro</field>
+            <field name="inherit_id" ref="account.view_partner_property_form"/>
+            <field name="model">res.partner</field>
+            <field name="arch" type="xml">
+                <field name="property_account_position" position="after">
+                    <field name="nrc" placeholder="e.g. J12/1234/2012" class="oe_inline"/>
                 </field>
```

Comment:
Even if the indentation is not very good, let's keep the diff minimal and do not modify the files where you are not changing the data